risc0/build_kernel/kernels/metal/fp.h

```diff
@@ -87,6 +87,9 @@ class Fp {
   /// Construct an FP from a uint32_t, wrap if needed
   constexpr Fp(uint32_t val) : val(encode(val)) {}
 
+  /// Construct an FP from a raw value
```

Comment:
Maybe clarify how this is different from `Fp(val)` (since "raw" might be interpreted as just "raw int not Fp object" not also "encoded in Montgomery form").
```suggestion
  /// Construct an Fp from an already-encoded raw value
```